code/modules/admin/verbs/smite.dm

```diff
@@ -0,0 +1,62 @@
+//lightning effect definition moved to holy.dm
+
+/client/proc/cmd_smite(mob/living/M in mob_list)
+	set category = "Fun"
+	set name = "Smite"
+	if(!holder)
+		usr << "no"
+		return
+
+	var/options = list("Brute","Burn","Toxin","Oxygen","Clone","Brain","Stamina","Heal","Gib","Cancel")
+	var/requiresdam = options - list("Heal","Gib","Cancel")
+
+	var/confirm = null
+	var/damtype_word = null
+	var/dam = 0
+
+	confirm = input(src, "Really smite [M.name]([M.ckey])?", "Divine Retribution") in list("Yeah", "Nah")
+	if(confirm == "Nah")
+		return
+
+	damtype_word = input(src, "What kind of damage?", "PUT YOUR FAITH IN THE LIGHT") in options
+
+	if(damtype_word in requiresdam)
+		dam = input(src, "How much damage?", "THE LIGHT SHALL BURN YOU") as num
```

Comment:
Why? this is adminbus there is no sanity here. Would this create a issue if I did not clamp it? Keep in mind this can be used on any mob I think not just humans I think you could use it on ghost too but it does not really do anything to them.
Also if they wanted they could just use variable edit to do like 1000000 burn damage, this way is just easier and more cool looking. There is probably already something that prevents humans from going over a certain damage limit like the procs it uses to do the damage.